package.json

```diff
@@ -37,7 +37,8 @@
     "rimraf": "~2.5.0",
     "tar": "~2.2.1",
     "tv4": "~1.2.7",
-    "swagger-tools": "~0.9.11"
+    "swagger-tools": "~0.9.11",
+    "moment": "~2.11.1"
```

Comment:
I just did experiment on this, I also see these package names are not alphabetized,  I remembered it should work in my previous testing, I see some people said about this: http://stackoverflow.com/questions/26934886/how-to-prevent-npm-install-package-save-dev-from-reordering-devdependencies

@zyoung51 : please ignore this comment, maybe my testing has something wrong. Alphabetical order or not will not impact function, so I'm OK with either way.